.github/renovate.json

```diff
@@ -22,15 +32,14 @@
       "groupName": "pyo3 non-major"
```

Comment:
@Hinton is this patching itself into some existing group? Otherwise I think we'd replace with the template.

Reply:
You'd need to specify the package names and update types to group on here though, which aren't present. Is this working as expected?

Reply:
Kind of, we just have to hold on updating until they all match. `"matchPackagePatterns": ["pyo3*"],`